C/include/c4Certificate.h

```diff
@@ -23,6 +23,8 @@
 
 #ifdef COUCHBASE_ENTERPRISE
 
+C4_ASSUME_NONNULL_BEGIN
```

Comment:
The assume end is outside the COUCHBASE_ENTERPRISE ifdef, causing CE builds to fail